spec/features/idv/analytics_spec.rb

```diff
@@ -967,143 +967,7 @@
-          timed_out: false,
-          transaction_id: nil,
-          review_status: 'pass',
-          account_lex_id: nil,
-          session_id: nil,
-        }
-      end
-
-      it 'records all of the events', allow_browser_log: true do
-        max_wait = Time.zone.now + 5.seconds
-        wait_for_event('IdV: user clicked what to bring link on ready to verify page', max_wait)
-        wait_for_event('IdV: user clicked sp link on ready to verify page', max_wait)
-        in_person_path_events.each do |event, attributes|
-          expect(fake_analytics).to have_logged_event(event, attributes)
-        end
-      end
-    end
-
-    # wait for event to happen
-    def wait_for_event(event, wait)
-      frequency = 0.1.seconds
-      loop do
-        expect(fake_analytics).to have_logged_event(event)
-        return
-      rescue RSpec::Expectations::ExpectationNotMetError => err
-        raise err if wait - Time.zone.now < frequency
-        sleep frequency
-        next
-      end
-    end
-  end
-
-  context 'Happy selfie path' do
-    before do
-      allow_any_instance_of(DocAuth::Response).to receive(:selfie_status).and_return(:success)
-
-      perform_in_browser(:desktop) do
-        sign_in_and_2fa_user(user)
-        visit_idp_from_sp_with_ial2(:oidc, facial_match_required: true)
-        complete_doc_auth_steps_before_document_capture_step
-        attach_images
-        attach_selfie
-        submit_images
-
-        click_idv_continue
-        visit idv_ssn_url
-        complete_ssn_step
-        complete_verify_step
-        fill_out_phone_form_ok('202-555-1212')
-        verify_phone_otp
-        complete_enter_password_step(user)
-        acknowledge_and_confirm_personal_key
-      end
-    end
-
-    it 'records all of the events' do
-      happy_mobile_selfie_path_events.each do |event, attributes|
-        expect(fake_analytics).to have_logged_event(event, attributes)
-      end
-    end
-
-    context 'proofing_device_profiling disabled' do
-      let(:proofing_device_profiling) { :disabled }
-      let(:threatmetrix) { false }
-      let(:threatmetrix_response_body) { nil }
-      let(:threatmetrix_response) do
-        {
-          client: 'tmx_disabled',
-          success: true,
-          errors: {},
-          exception: nil,
-          timed_out: false,
-          transaction_id: nil,
-          review_status: 'pass',
-          account_lex_id: nil,
-          session_id: nil,
-        }
-      end
-
-      it 'records all of the events' do
-        aggregate_failures 'analytics events' do
-          happy_mobile_selfie_path_events.each do |event, attributes|
-            expect(fake_analytics).to have_logged_event(event, attributes)
-          end
-        end
-      end
-    end
-  end
-  context 'doc_auth_separate_pages_enabled is true' do
-    before do
-      allow(IdentityConfig.store).to receive(:doc_auth_separate_pages_enabled).and_return(true)
-    end
+  context 'split doc auth is true' do
```

Comment:
i don't think we need this context anymore since this is the default and unchanging behavior 🤔